bdio2/src/main/java/com/blackducksoftware/bdio2/BdioMetadata.java

```diff
@@ -146,6 +145,27 @@ public BdioMetadata merge(Map<String, Object> other) {
                     // Incompatible identifiers
                     throw new IllegalArgumentException("identifier mismatch: " + value + " (was expecting " + id() + ")");
                 }
+            }  else if (key.equals(JsonLdConsts.TYPE)) {
+                Bdio.ScanType scanType;
+
+                // When a Jsonld is processed the library adds any other metadata other than id into a list
+                if(value instanceof List<?> && !((List<?>) value).isEmpty()){
+                    scanType = Bdio.ScanType.from(((List<?>) value).get(0));
```

Comment:
I guess this opens up an interesting question.   Is it unreasonable to anticipate supporting multiple scan types in a single document?   Can the specification leave an open door to support this even if we don't support it initially within the Black Duck consumption model?

Reply:
I am not sure, but the current spec doesn't allow multiple scan types in a single document right ?.

Reply:
Yes, and while we know this is coming (Detect team has tickets for it), Agile would tell us to wait and refactor when it happens - right?

Reply:
Does this change allow us to support that in the future without introducing a backwards-compatibility breaking change?

Reply:
Yes, I would agree with Joe here. Also, on the implementation side, we do have the capability to support both at the same time but as currently, we are not doing so, we are keeping this CAP to keep things clear.